docs/markdown/Style-guide.md

```diff
@@ -13,9 +13,14 @@ Always spaces.
 
 ## Naming Variable
 
-The most consistent naming convention is the snake case. Let say you would
-like to refer to your executable so something like `my_exe` would work or
-just `exe`.
+Snake case (stylized as `snake_case`) refers to the style of writing in which
+each space is replaced by an underscore (`_`) character, and the first letter of
+each word written in lowercase. It is the most common naming convention used
+in Meson build scripts as identifiers for variable.
+
+Let say you would like to refer to your executable so something like `my_exe`.
+
+Developers tend to recognize snake case values more quickly than camel case.
```

Comment:
FWIW I think this sentence is hard to justify and also unnecessary. A style guide doesn't have to be anything but subjective guidance.